src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,34 @@
+import { fetchArticles } from "../../utils/api";
+import { useEffect, useState } from "react";
+import ArticleLink from "./ArticleLink";
+
+export default function ArticlesList() {
+  const [articles, SetArticles] = useState([]);
```

Comment:
the standard here is camelCase so setArticles instead

Reply:
OK - updated